lms/services/jwt.py

```diff
@@ -75,7 +76,9 @@ def decode_lti_token(self, id_token: str) -> dict:
 
         if not jwt.get_unverified_header(id_token).get("kid"):
             LOG.debug("Missing 'kid' value in JWT header")
```

Comment:
Double-check this but I think you might be able to remove these `LOG.debug()` calls now because raised exceptions get logged automatically

Reply:
The exception doesn't show up on the logs locally. I'll check in paper-trail when deployed